src/plugin.js

```diff
@@ -226,6 +228,29 @@ const onPlayerReady = (player, emeError) => {
       setupSessions(player);
       handleWebKitNeedKeyEvent(event, getOptions(player), player.tech_)
         .catch(emeError);
+    };
+
+    // Support Safari EME with FairPlay
+    // (also used in early Chrome or Chrome with EME disabled flag)
+    player.tech_.el_.addEventListener('webkitneedkey', (event) => {
+      // It's possible that at the start of playback a rendition switch
+      // on a small player in safari's HLS implementation will cause
+      // two webkitneedkey events to occur. We want to make sure to cancel
+      // our first existing request if we get another within 1 second. This
+      // prevents a non-fatal player error from showing up due to a
+      // request failure.
+      if (!player.eme.webkitneedkey.first) {
+        player.clearTimeout(player.eme.webkitneedkey.timeout);
+        player.eme.webkitneedkey.timeout = player.setTimeout(() => {
+          player.eme.webkitneedkey.first = true;
+          player.eme.webkitneedkey.timeout = null;
+          handleFn(event);
+        }, 1000);
```

Comment:
So, this means that safari+fairplay will always take at least 1s to start playback? Should we make this value configurable?

Reply:
I will add a configuration option but default it to 1000ms. How does that sound?

Reply:
Sounds fine to me. Was there a smaller value that you found worked or is 1s the most reliable you've found?